index.html

```diff
@@ -665,7 +665,7 @@ <h4>Basic usage</h4>
       </div>
 
       <div class="docs">
-        <pre>
+        <pre><code>
```

Comment:
It looks like this should be a closing tag rather than an opening which is causing some issues for everything below this

![image](https://user-images.githubusercontent.com/13633775/33621403-a5ea925c-d9b8-11e7-90ae-714bd33f49be.png)

Reply:
Oops! Thanks for pointing